src/table_plane_extractor_srv.py

```diff
@@ -97,8 +77,10 @@ def table_plane_extractor_server():
     Topic: /test/table_plane_extractor
     '''
     rospy.init_node('table_plane_extractor_server')
-    s = rospy.Service('/test/table_plane_extractor',
-                      TablePlaneExtractor, table_plane_extractor_methode)
+    s = rospy.Service(
+        '/test/table_plane_extractor',
```

Comment:
We should probably also update this service name. Maybe /table_plane_extractor/get_planes or something like that?

Reply:
Dont forget to update the readme as well.